config/validate/rules.go

```diff
@@ -29,6 +32,9 @@ type rule func(config node, report *Report)
 var Rules []rule = []rule{
 	checkStructure,
 	checkValidity,
+	checkWriteFiles,
+	checkDiscoveryUrl,
```

Comment:
imho just sort lexicographically then, it makes it slightly clearer there is no other significance